lib/failhandler/failhandler.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2014 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package failhandler
```

Comment:
This seems like it could be in the usage reporting package rather than its own thing?

Reply:
If we treat it like usage reporting, then sure, otherwise it feels weird having them in the same place.